### DISCUSSION TITLE
Change ride profit per hour to ride profit per month

### DISCUSSION THREAD

**Discussion by HaasJona** [OpenRCT2/OpenRCT2#15716] [Feature requests]
Ride profit is currently shown per hour of play time (or more correctly 115200 game update ticks), which is not really useful anymore as you can speed up the game and also it's not very accurate. I think it would be better expressed as profit per ingame month which can be accurately expressed as a fixed number of ticks.

**Comment by Gymnasiast** [comment]
A good idea. An alternative would be to have the profit per year, but I have no strong leaning towards either.

**Comment by janisozaur** [comment]
I'd rather go with a monthly one, that sounds like a good idea to me too.

Does the historical value of guest count get stored in `sv6`?

**Comment by HaasJona** [comment]
> I'd rather go with a monthly one, that sounds like a good idea to me too.

I would also prefer monthly to yearly.

> Does the historical value of guest count get stored in sv6?

Yes, the 10 last 30 second (960 game tick) intervals are stored starting at 0x124. Then these values are added and multiplied by 12 to get the hourly value currently. See #2561

**Comment by pizza2004** [comment]
I was just talking to my friend about this the other day.  Monthly would be the most useful from a gameplay standpoint.

**Comment by wisepotato** [comment]
How about making this an option? Let the user decide what they find comfortable: monthly in game, hourly irl, yearly in game, by minute irl. Seems the best option to me.

**Comment by IntelOrca** [comment]
@wisepotato Literally everyone has said monthly is fine and those who haven't probably aren't bothered enough to care. So probably unnecessary.

**Comment by Ryder17z** [comment]
Monthly would be very useful

**Comment by IntelOrca** [comment]
I think we can end the discussion now, as nearly everyone is agreeing with monthly.

**Comment by HaasJona** [comment]
Okay. I will implement this when I have some spare time.

**Comment by wisepotato** [comment]
@IntelOrca was just thinking that maybe an option would make it possible for everybody to be happy that ever played the game, a suggestion is just that; a suggestion :smile:

**Comment by IntelOrca** [comment]
@wisepotato how about we wait for someone to passionately ask for a year amount before going down that road...
